### PR TITLE
Updated Docker and other files including result folder fixes

### PR DESCRIPTION
**Pull request by kaushik691**
Modified docker code, still need to work on bulk loader for docker

### CODE REVIEW

File: resources/dgraph_commands.robot [hunk 1]
```diff
@@ -668,12 +681,16 @@ Check if parallel process is triggered
 
 
 Backup alpha and zero logs
-    [Documentation]
+    [Documentation]     Kewyword to backup all the logs
```
Comment (souravDgraph) [inline resources/dgraph_commands.robot:684]: typo.

File: test_suites/dgraph/docker/dgraph_docker_suite.robot [hunk 2]
```diff
@@ -24,5 +23,47 @@ TC_01 Docker - Verify Increment Operation
...
+     [Tags]    regression   C702    C700   WEEKLY
+     Clear Backup Folders   true
+     Create NFS Backup      1
+     log        ${is_latest_global_check}
+     Run Keyword If     ${is_latest_global_check}     Perform a restore on backup latest versions    0
+     ...    ELSE    Perform a restore on backup by older dgraph versions
+     Clear Backup Folders   true
+     [Teardown]    NONE
+
+#TC_05 Docker - Perform parallel live and bulk load on data
```
Comment (souravDgraph) [inline test_suites/dgraph/docker/dgraph_docker_suite.robot:47]: un comment.

File: test_suites/dgraph/Linux/dgraph_suite_01.robot [hunk 3]
```diff
@@ -63,5 +66,8 @@ TC_06 Perform Increment backup and restore data
 TC_07 Perform parallel live loads.
      [Documentation]    Perform live load data.
      ...    *Author*: Sourav
-     [Tags]    regression   WEEKLY  CI
+     [Tags]    regression   WEEKLY
      Execute Multiple Parallel Live Loader with rdf and schema parameters    ${rdf_file}    ${schema_file}    2
+     [Teardown]    NONE
+
```
Comment (souravDgraph) [inline test_suites/dgraph/Linux/dgraph_suite_01.robot:72]: remove extra lines.

File: resources/dgraph_commands.robot [hunk 4]
```diff
@@ -249,12 +254,24 @@ Trigger Loader Process
     ${conf_loder_command}=    Get Dgraph Loader Command    ${path}/test_data/datasets/${rdf_filename}    ${path}/test_data/datasets/${schema_filename}       ${loader_name}     is_latest_version=${is_latest_global_check}  docker_string=${docker_exe_string}      
     ${result_loader}=   Process.start Process    ${conf_loder_command}    alias=${loader_alias}    stdout=${loader_alias}.txt    stderr=${loader_alias}_err.txt    shell=True    cwd=results
 
+Verify Live loader trigger properly or not
+    [Documentation]  Keyword to verify live loader to trigger properly
+    [Arguments]  ${loader_alias}    ${rdf_filename}    ${schema_filename}     ${loader_name}
+    ${status}   Run Keyword And Return Status   Wait Until Keyword Succeeds    3x    10 sec    Grep and Verify file Content in results folder    ${loader_alias}    N-Quads:
```
Comment (souravDgraph) [inline resources/dgraph_commands.robot:260]: 30secs is less, please make it 60 secs.

File: resources/dgraph_commands.robot [hunk 5]
```diff
@@ -249,12 +254,24 @@ Trigger Loader Process
...
+    IF  ${status}==${FALSE}
+        ${result_check}=    Run Keyword And Return Status   Wait Until Keyword Succeeds    2x    10 sec    Grep and Verify file Content in results folder    ${loader_alias}    Please retry
+        Run Keyword If    ${result_check}        Monitor Live loader Process     ${loader_alias}     ${rdf_filename}    ${schema_filename}     ${loader_name}
+        ...     ELSE    FAIL    Some issue with live loader
+    END
+    #Run Keyword And Return If    "${result_check}" == "True"    Fail    Error while processing schema file
```
Comment (souravDgraph) [inline resources/dgraph_commands.robot:266]: can be removed.

File: resources/dgraph_commands.robot [hunk 6]
```diff
@@ -167,10 +171,10 @@ Post Execution Verify Alpha contents
     @{dir}    Create List    p   t
     @{alpha_err_context}  Create List     Dgraph Version  Dgraph codename
     ${passed}=  Run Keyword And Return Status   Wait Until Keyword Succeeds     5x    5 sec   Verify alpha and zero contents in results folder    alpha    @{alpha_err_context}
-    Run Keyword And Return If      ${passed}==${FALSE}       Fatal Error
+    Run Keyword And Return If      ${passed}==${FALSE}       Fail   alpha Initlization failed.
```
Comment (souravDgraph) [inline resources/dgraph_commands.robot:174]: typo.

File: resources/dgraph_commands.robot [hunk 7]
```diff
@@ -167,10 +171,10 @@ Post Execution Verify Alpha contents
...
     @{alpha_error_context}  Create List     Error: unknown flag     panic: runtime error:   runtime.goexit
     ${passed}=  Run Keyword And Return Status   Wait Until Keyword Succeeds     5x    5 sec   Verify alpha and zero contents in results folder    alpha    @{alpha_error_context}
-    Run Keyword And Return If      ${passed}       Fail
+    Run Keyword And Return If      ${passed}       Fail     Captured few errors in aplpha
```
Comment (souravDgraph) [inline resources/dgraph_commands.robot:177]: typo.

**Review by souravDgraph — COMMENTED** [review]
Some minor comments please address before pushing.

File: resources/dgraph_commands.robot (hunk 7 above)
Comment (kaushik691) [reply resources/dgraph_commands.robot:177]: okay

File: resources/dgraph_commands.robot (hunk 6 above)
Comment (kaushik691) [reply resources/dgraph_commands.robot:174]: okay

File: resources/dgraph_commands.robot (hunk 5 above)
Comment (kaushik691) [reply resources/dgraph_commands.robot:266]: okay

File: resources/dgraph_commands.robot (hunk 4 above)
Comment (kaushik691) [reply resources/dgraph_commands.robot:260]: okay

File: test_suites/dgraph/Linux/dgraph_suite_01.robot (hunk 3 above)
Comment (kaushik691) [reply test_suites/dgraph/Linux/dgraph_suite_01.robot:72]: okay

File: test_suites/dgraph/docker/dgraph_docker_suite.robot (hunk 2 above)
Comment (kaushik691) [reply test_suites/dgraph/docker/dgraph_docker_suite.robot:47]: for now i didn't fix for bulk loader

File: resources/dgraph_commands.robot (hunk 1 above)
Comment (kaushik691) [reply resources/dgraph_commands.robot:684]: okay

**Pull request merged**